### ISSUE TITLE
About training details

### ISSUE THREAD

**Issue by zhang-haojie** [Vchitect/Latte#110]
I noticed in the previous issue that it will reach convergence after about 150,000 iters. Does this refer to the training results when the batchsize is 5? 

I checked the training code and the number of iterations here is independent of the number of GPUs, right?

**Comment by maxin-cn** [comment]
Based on other people's replications, the performance of the number of iterations is related to the number of GPUs used. More details can be found in this [issue](https://github.com/Vchitect/Latte/issues/104).

**Comment by zhang-haojie** [comment]
But in theory, one GPU' train_step will be counted as one iteration. I guess the number of GPUs affects the batch of gradient backpropagation in DDP strategy.
